### PR TITLE
Jonmv/restore json deps test scope

### PR DESCRIPTION
**Pull request by jonmv**
@gjoranv

### CODE REVIEW

**Comment by bratseth** [comment]
Could we put a `// TODO: Remove on Vespa 9 `on these?

**Comment by jonmv** [comment]
That would be nice idneed.

File: container-test/pom.xml [hunk 1]
```diff
@@ -100,6 +100,12 @@
       <artifactId>commons-compress</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <!-- not used by Vespa, but was historically on test classpath -->
+      <groupId>org.json</groupId>
+      <artifactId>json</artifactId>
+      <scope>compile</scope>
```
Comment (gjoranv) [inline container-test/pom.xml:107]: Please don't use explicit 'compile' scope. It could imply that we manage dependency scopes, which we shouldn't.

**Review by gjoranv — APPROVED** [review]

File: container-test/pom.xml (hunk 1 above)
Comment (jonmv) [reply container-test/pom.xml:107]:
```suggestion
```

File: container-test/pom.xml [hunk 2]
```diff
@@ -100,6 +100,11 @@
       <artifactId>commons-compress</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <!-- not used by Vespa, but was historically on test classpath -->
```
Comment (gjoranv) [inline container-test/pom.xml:104]: Please add Vespa 9 TODO here as well.

File: parent/pom.xml [hunk 3]
```diff
@@ -994,6 +994,11 @@
                 <artifactId>xercesImpl</artifactId>
                 <version>2.12.2</version>
             </dependency>
+            <dependency>
```
Comment (gjoranv) [inline parent/pom.xml:997]: Please add Vespa 9 TODO here as well.

**Pull request merged**